player/lua/auto_profiles.lua

```diff
@@ -40,6 +43,20 @@ local function evaluate(profile)
     profile.dirty = false
 end
 
+local function guess_invalid_property(name)
+    local toplevel_property = nil
+    for property in name:gmatch("([^/]+)") do
```

Comment:
with `match()` instead of `gmatch()` you could avoid the loop

Reply:
Ended up code golfing it but with some comments.